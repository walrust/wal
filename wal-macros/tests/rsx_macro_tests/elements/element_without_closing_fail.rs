use wal_macros::rsx;

fn main() {
    rsx! { <div> };
    rsx! { <div> "Hello world!" };
    rsx! { <div> { String::from("Hello world") } };
    rsx! { <div><div></div> };
}
