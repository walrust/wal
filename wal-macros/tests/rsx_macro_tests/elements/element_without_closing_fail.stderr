error: This start tag does not have a corresponding end tag. (hint: try adding `</div>`)
 --> tests/rsx_macro_tests/elements/element_without_closing_fail.rs:4:12
  |
4 |     rsx! { <div> };
  |            ^^^^^

error: This start tag does not have a corresponding end tag. (hint: try adding `</div>`)
 --> tests/rsx_macro_tests/elements/element_without_closing_fail.rs:5:12
  |
5 |     rsx! { <div> "Hello world!" };
  |            ^^^^^

error: This start tag does not have a corresponding end tag. (hint: try adding `</div>`)
 --> tests/rsx_macro_tests/elements/element_without_closing_fail.rs:6:12
  |
6 |     rsx! { <div> { String::from("Hello world") } };
  |            ^^^^^

error: This start tag does not have a corresponding end tag. (hint: try adding `</div>`)
 --> tests/rsx_macro_tests/elements/element_without_closing_fail.rs:7:12
  |
7 |     rsx! { <div><div></div> };
  |            ^^^^^
